tests/queries/0_stateless/01356_initialize_aggregation.sql

```diff
@@ -1,4 +1,5 @@
 SELECT uniqMerge(state) FROM (SELECT initializeAggregation('uniqState', number % 3) AS state FROM system.numbers LIMIT 10000);
 SELECT topKWeightedMerge(10)(state) FROM (SELECT initializeAggregation('topKWeightedState(10)', number, number) AS state FROM system.numbers LIMIT 1000);
 SELECT topKWeightedMerge(10)(state) FROM (SELECT initializeAggregation('topKWeightedState(10)', 1, number) AS state FROM system.numbers LIMIT 1000);
-SELECT topKWeightedMerge(10)(state) FROM (SELECT initializeAggregation('topKWeightedState(10)', number, 1) AS state FROM system.numbers LIMIT 1000);
+-- order in libstdc++ and libc++ may be a little different, that's why we need arraySort()
+SELECT arraySort(topKWeightedMerge(10)(state)) FROM (SELECT initializeAggregation('topKWeightedState(10)', number, 1) AS state FROM system.numbers LIMIT 1000);
```

Comment:
The following patch will have the same effect:

```patch
diff --git a/src/Common/SpaceSaving.h b/src/Common/SpaceSaving.h
index 5606334024..0f148568da 100644
--- a/src/Common/SpaceSaving.h
+++ b/src/Common/SpaceSaving.h
@@ -241,7 +241,7 @@ public:
             }
         }
 
-        std::sort(counter_list.begin(), counter_list.end(), [](Counter * l, Counter * r) { return *l > *r; });
+        std::stable_sort(counter_list.begin(), counter_list.end(), [](Counter * l, Counter * r) { return *l > *r; });
 
         if (counter_list.size() > m_capacity)
         {
```

Reply:
Ok, but we don't actually need stable sort.

Reply:
Indeed, that's why arraySort